app/src/main/res/values/donottranslate_questionnaire_values.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Sorry, I'm not sure I understand your question.
I think it's better way to use `strings.xml` to translate.
Like this.

### arrays_questionnaire.xml
```xml
    <string-array name="questionnaire_values_age">
        <item>@string/questionaire_item_10_age</item>
    </string-array>
```

If you choose this way, it's enough to use only `arrays_questionnaire.xml`.